src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -363,6 +363,8 @@ void Terminal::_WriteBuffer(const std::wstring_view& stringView)
     auto& cursor = _buffer->GetCursor();
     const Viewport bufferSize = _buffer->GetSize();
 
+    cursor.StartDeferDrawing();

```

Comment:
Leave behind the comment explaining why this is required (for performance reasons, we can not waste time displaying a cursor event when we know more text is coming right behind it).